app/screens/reaction_list/reaction_header.js

```diff
@@ -41,15 +43,19 @@ export default class ReactionHeader extends PureComponent {
 
     render() {
         return (
-            <View style={styles.container}>
-                <ScrollView
-                    alwaysBounceHorizontal={false}
-                    horizontal={true}
-                    overScrollMode='never'
-                >
-                    {this.renderReactionHeaderItems()}
-                </ScrollView>
-            </View>
+            <NativeViewGestureHandler
+                ref={this.props.forwardedRef}
+            >
+                <Animated.View style={styles.container}>
+                    <ScrollView
```

Comment:
Could this use `Animated.ScrollView`? 0/5 since I don't really understand what that component does differently

Reply:
`Animated.ScrollView` will wrap it in another HOC component and is not actually needed here